superchain/README.md

```diff
@@ -13,7 +13,8 @@ SDK             | Version
 `OpenJDK 8`     | Amazon Corretto `>= 8.242.08.1`
 `.NET SDK`      | `>= 3.1.101`
 `mono`          | `>= 6.8.0.105`
-`Javascript`    | `node >= 10.19.0` with `npm >= 6.13.4`
+`Javascript`    | `node >= 10.19.0` with `npm >= 6.13.4` (default)
+                | OR `node >= 14.16.0` with `npm >= 6.14.11` (`--build-arg NODE_VERSION=14`)
```

Comment:
This won't render properly... Seems like the rendering engine does not like when the first column is empty.

Also - you don't want to make this relative to `--build-arg`, instead refer to the tag names.

Perhaps we should actually extract the `node` and `npm` info to a sub-section - and in the table mention this is tag-dependent, and link to it?